src/core/client/framework/lib/relay/useLoadMore.ts

```diff
@@ -4,23 +4,29 @@ import { RelayPaginationProp } from "react-relay";
 /**
  * useLoadMore is a react hook that returns a `loadMore` callback
  * and a `isLoadingMore` boolean.
+ *
+ * @param relay {RelayPaginationProp}
+ * @param count {number}
  */
 export default function useLoadMore(
   relay: RelayPaginationProp,
   count: number
-): [() => void, boolean] {
+): [() => Promise<void>, boolean] {
```

Comment:
is this related or is this just an unrelated bugfix?

Reply:
It looks like it's related to collecting the start/finish of load more events.